### DISCUSSION TITLE
무료 제공 소프트웨어와 서비스

### DISCUSSION THREAD

**Discussion by RangHo** [RangHo/sogang-courses#13] [꿀팁게시판]
서강대학교에서는 학생을 위해 여러 소프트웨어와 서비스를 무료로 제공합니다. 또, 학생 신분이면 무료로 사용할 수 있는 서비스도 많이 있어요. 이 포스트에서는 학교에서 제공하는 서비스와 외부에서 제공하는 무료 서비스를 모읍니다.

가난한 대학생의 지갑을 살려주세요!

**Comment by RangHo** [comment]
# 학교 이메일 주소

학교 이메일 주소는 아마 대부분의 학생들이 처음으로 받는 **공적인** 이메일 주소일 거예요. 만약 논문을 쓰거나 교환학생을 가는 등, 학교의 학생으로서 활동을 한다면 이 학교 이메일 주소를 사용하게 됩니다. 그러니 이메일 계정을 만들 때 이메일 아이디는 장난스럽게 지으시면 안 돼요!

이메일 아이디는 되도록 이름과 성을 조합해서 만들되, 생일 등의 숫자는 빼 주세요. 기억에 남는 닉네임이나 단어를 사용해도 괜찮습니다. 대외적으로 공개되어도 문제가 없는 선에서이지만요.

## 가입 절차

1. [SAINT](https://saint.sogang.ac.kr/)에 로그인
2. 페이지 중간에 있는 `E-Mail` 버튼 클릭
3. 회원가입 클릭

회원가입 후, 관리자 승인이 필요하니 잠시 기다려 주세요.

**Comment by RangHo** [comment]
# Microsoft Office 365

학교 아이디를 통해 Microsoft Office를 사용할 수 있습니다. ~~Google Drive는 10GB로 용량이 제한되지만 OneDrive는 그 제한이 없으므로 대용량 파일 공유가 필요하다면 OneDrive를 이용하세요.~~ 2024년 2월 기준, OneDrive도 10GB로 제한됩니다. 참고로, 학교에서 발급해 주는 Microsoft 계정으로 접속해야 사용 가능하다는 점 주의하세요!

## 가입 절차

1. [서강대학교 U-cloud](https://ucloud.sogang.ac.kr/) 로그인 (SAINT ID와 비밀번호 사용)
2. 로그인 후 우측 아이콘 중 세 번째 아이콘 클릭하여 오피스 다운로드

## 제공하는 서비스

* Microsoft Office 365 구독
* Outlook 이메일 주소 (`<username>@o365.sogang.ac.kr`)
* ~~무제한~~ 10GB OneDrive 저장공간

**Comment by RangHo** [comment]
# Google Workspace

Microsoft Office와 마찬가지로 학교 이메일을 사용해서 Google Workspace를 사용할 수 있습니다. 2021년까지는 무제한 저장공간을 주었지만 2022년부터 [Google이 교육기관에 대한 무료 저장공간 제공을 종료하면서](https://blog.google/outreach-initiatives/education/google-workspace-for-education/) 10GB로 제한이 들어갑니다. 대용량 파일 공유는 OneDrive를 활용하세요.

특히 안드로이드 기반 폰을 사용하고 계신다면 이 학교 계정으로 동기화도 가능하니, 학교 프로필과 개인 프로필을 분리하고 싶은 분이라면 활용하셔도 좋을 것 같아요.

## 가입 절차

1. [서강대학교 U-cloud](https://ucloud.sogang.ac.kr/) 로그인 (SAINT ID와 비밀번호 사용)
2. 로그인 후 가운데 메뉴에서 Gmail이나 Drive 아이콘 클릭

## 제공하는 서비스

* Gmail 이메일 주소 (`<username>@u.sogang.ac.kr`)
* 10GB Google Drive 저장공간 (공유 드라이브 사용 가능)
* Google Classroom

**Comment by RangHo** [comment]
# GitHub Student Developer Pack

바로 이 사이트, GitHub에서 제공하는 학생 개발자 플랜입니다. 무료로 GitHub Pro를 사용하실 수 있다는 게 가장 큰 장점이네요. 얼마 전부터 GitHub 무료 계정도 비공개 리포지토리를 팔 수 있게 되면서 조금 메리트는 적어졌지만요. 그래도 킬러 기능은 남아 있으니 꼭 신청해서 사용해 보세요!

## 가입 절차

1. [GitHub 신청 페이지](https://education.github.com/discount_requests/application)에 접속
2. 학교 이메일 등록이 되어 있지 않다면 `+ Add an email address` 버튼을 클릭해서 학교 이메일 등록
3. 학교 이메일을 선택한 뒤 학교 이름이 정확한지 확인
4. 주어진 칸에 GitHub 사용 용도를 간결하게 영어로 작성 후 다음 단계로 진행
5. 가지고 있는 학생증의 앞면을 찍어서 업로드 후, 종류는 `Student ID`로 선택
6. 가입 신청 후 승인 대기

영작이 익숙하지 않으신 분들은 그냥 GitHub에서 무슨 일을 하는지 간단하게 리스트업만 하셔도 됩니다!

## 제공하는 서비스

* GitHub Pro
* __GitHub Copilot__ (중요)
* GitHub Codespaces Pro
* GitLens+ Pro
* GitHub Campus Experts 지원 가능
* [기타등등...](https://education.github.com/pack/offers)

**Reply by tkdgus0782** [reply]
모바일 학생증밖에 없어서 안돼요 ㅠㅠ

**Reply by RangHo** [reply]
@tkdgus0782 영문 재학증명서 뽑아서 스캔 후 제출해도 문제가 없습니다... 그래도 요즘 학생증은 마스터카드도 지원하는 모양이니 이번 기회에 재발급을...

**Comment by RangHo** [comment]
# ~~Windows 10/11 Education 라이센스~~ 서강대학교 OnTheHub 페이지

~~학생 신분인 동안 Windows 10과 11 라이센스를 무료로 받아 사용할 수 있습니다! [학교 이메일 주소](https://github.com/RangHo/sogang-courses/discussions/13#discussioncomment-5603105)와 [Microsoft Office 365](https://github.com/RangHo/sogang-courses/discussions/13#discussioncomment-5603161) 계정이 먼저 필요하니 이 쪽 먼저 만들어 주세요. 참고로 처음 발급받은 라이센스 키를 잃어버리면 돈을 내고 복구를 해야 하니, 받자마자 어딘가에 잘 적어 두셔야 합니다!~~

2023년 11월 기준 Windows 라이센스 항목이 사라졌습니다!

그 외에 다른 소프트웨어는 아직 팔고 있긴 한데, 일반적으로 쓸 수 있는 무료 프로그램은 좋은 게 보이지가 않네요...

## 가입 절차

> 자세한 설명은 아래 링크에! \
> http://www.ssodam.com/content/1585474

1. [서강대학교 OnTheHub 페이지](https://sogang.onthehub.com) 접속
2. 오른쪽 `등록` 버튼을 눌러서 계정 생성
3. 정보 입력 후 학교 이메일에 접속해서 자격 증명 메일 확인

## 제공하는 서비스

여러가지 소프트웨어를 할인된 가격에 파는데, 쓸 만 한 것들만 적어보겠습니다.

* ~~__Windows 10/11 Education 라이센스 (무료)__~~
* Maxon One (Cinema 4D, Red Giant, Redshift, Universe, Zbrush) 6개월 라이센스 (무료/할인)
* ~~AppGameKit Ultimate Education Pack (할인)~~
* ~~NordVPN, NordPass 구독 (할인)~~

**Comment by RangHo** [comment]
# MATLAB

전자과의 애증을 한몸에 받는 친구, MATLAB입니다. 일단은 MATLAB 뿐만 아니라 스토리지로도 쓸 수 있는 플랫폼이긴 한데요... OneDrive가 있는 시점에 굳이 여기를 써야 할까요? 이것도 [학교 이메일 주소](https://github.com/RangHo/sogang-courses/discussions/13#discussioncomment-5603105)가 필요하기 때문에 먼저 만들어 주세요.

## 가입 절차

1. 서강대학교 [캠퍼스 클라우드](https://scloud.sogang.ac.kr/) 접속
2. SAINT 아이디로 로그인
3. 웹 탐색기로 들어가서 `공통소프트웨어 > 3.소프트웨어(학생용) > 3.MATLAB` 폴더 확인
4. 설치 가이드 PDF 파일을 다운받아 가입 절차 확인

## 제공하는 서비스

* MATLAB 학생 라이센스
* ~~캠퍼스 클라우드 스토리지~~

**Comment by RangHo** [comment]
# Notion Education Plus

노트부터 간단한 웹 호스팅까지, 생산성 관리 툴의 최강자 Notion 플러스 요금제를 무료로 사용할 수 있습니다. 학교 이메일 계정이 남아있는 한 계속 무제한으로 사용할 수 있으니 꼭 신청해 보세요!

## 가입 절차

1. [교육을 위한 Notion](https://www.notion.so/ko-kr/product/notion-for-education) 페이지 접속
2. 학교 이메일 주소 이용해서 회원가입

## 제공하는 서비스

* Notion Plus 기능

**Comment by RangHo** [comment]
# Adobe Creative Cloud Education

컨텐츠 제작의 필수품인 Adobe Creative Cloud를 학생 할인가에 구매할 수 있습니다! 1년간만 할인된 가격으로 쓸 수 있고, 그 뒤에는 일반 가격이 적용된다고 해요. 해지 시 위약금 달라고 깽판 치기로 유명한 플랜이라 해지할 때는 위약금 청구 기간 잘 맞추셔야 할 수도 있어요. 1년 사용하기로 약정이 걸려 있는 거라서요...

## 가입 절차

1. [Adobe Creative Cloud Education](https://www.adobe.com/kr/creativecloud/plans.html?plan=edu) 페이지 접속
2. `구매하기` 버튼 클릭 후 결제 진행

## 제공하는 서비스

* Adobe Creative Cloud 소프트웨어 전체

**Comment by RangHo** [comment]
# JetBrains Education License

Java 계열 언어용 IDE의 최강자, JetBrains의 IDE와 서비스를 학생 신분인 동안 무료로 사용할 수 있습니다! IntelliJ는 물론이고, ReSharper같은 서비스도 사용할 수 있어요.

## 가입 절차

1. [JetBrains Products for Learning 페이지](https://www.jetbrains.com/shop/eform/students) 접속
2. 학생 인증
    1. GitHub Education 인증을 완료한 경우, 이를 이용해서 학생 인증이 가능
    2. 학교 이메일을 이용해서 인증
    3. ISIC(국제학생증) 카드 번호로 인증
3. `APPLY FOR FREE PRODUCTS` 버튼 클릭

## 제공하는 서비스

* IntelliJ IDEA Ultimate
* ReSharper, ReSharper C++
* dotPeek, dotTrace, dotMemory, dotCover
* IntelliJ 기반 에디터들 (RubyMine, PyCharm, PhpStorm 등...)
